Makefile

```diff
@@ -1,80 +1,35 @@
-.PHONY: install
-install:
-	poetry install
-
-.PHONY: setup-remote-state
-setup-remote-state:
-	poetry run dotenv run bash ./scripts/setup_remote_state.sh
-
-.PHONY: destroy-remote-state
-destroy-remote-state:
-	poetry run dotenv run sh -c 'az group delete --resource-group $$TF_VAR_identifier-bakery-remote-state-resource-group --yes'
-
 .PHONY: init
 init:
-	poetry run dotenv run terraform -chdir="terraform/" init
-
-.PHONY: lint-init
-lint-init:
-	terraform -chdir="terraform/" init -backend=false
-
-.PHONY: lint
-lint: lint-init
-	terraform -chdir="terraform/" validate
-	poetry run flake8 test/recipes/ scripts/
-	poetry run isort --check-only --profile black test/recipes/ scripts/
-	poetry run black --check --diff test/recipes/ scripts/
-
-.PHONY: format
-format:
-	terraform -chdir="terraform/" fmt
-	poetry run isort --profile black test/recipes/ scripts/
-	poetry run black test/recipes/ scripts/
+	./scripts/init.sh
 
-.PHONY: plan
-plan: init
-	poetry run dotenv run terraform -chdir="terraform/" plan
-
-.PHONY: apply
-apply: init
-	poetry run dotenv run terraform -chdir="terraform/" apply -auto-approve
+.PHONY: install
+install:
+	./scripts/install.sh
 
 .PHONY: destroy
-destroy: init
-	poetry run dotenv run terraform -chdir="terraform/" destroy -auto-approve
-
-.PHONY: configure-kubectl
```

Comment:
With this target removed how am I supposed to use Lens to connect to the cluster?  Can we include some Lens and kubectl access docs in the README.  Many people who will deploy these clusters will have minimal k8s experience and will need a guide.